playground/angular/src/app/app.component.html

```diff
@@ -0,0 +1,20 @@
+<div class="container">
+  <atropos
+    class="atropos-banner"
+    [highlight]="false"
+    [rotate]="rotate"
```

Comment:
@nolimits4web any idea why it's stopped working?
I'm updating atroposref.params on params changes